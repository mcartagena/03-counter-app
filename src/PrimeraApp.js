import React from "react";
//import React, {Fragment} from "react";

const PrimeraApp = ({saludo = 'Hola Mundo'}) => {

    /* const saludo = 'Hola Mundo';

    console.log(props);

     const objeto = {
        nombre: 'Marcelo',
        edad: 51
    }; */

    return (
        <>
            <h1>{saludo}</h1>
            {/*<pre>{JSON.stringify(objeto, null, 3)}</pre>*/}
            <p>Mi primera aplicacion.</p>
        </>
    );
}

export default PrimeraApp;