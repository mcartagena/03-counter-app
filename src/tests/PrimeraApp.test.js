import React from "react";
import {shallow} from 'enzyme';

import PrimeraApp from "../PrimeraApp";

describe('Pruebas en <PrimeraApp />', () => {

    // test('debe mostrar el mensaje "Hola, soy Marcelo"', () => {
    //
    //     const saludo = 'Hola, soy Marcelo';
    //
    //     const {getByText} = render(<PrimeraApp saludo={saludo}/>);
    //
    //     expect(getByText(saludo)).toBeInTheDocument();
    //
    // });

    test('debe mostrar <PrimeraApp /> correctamente', () => {

        const saludo = 'Hola, soy Marcelo';
        const wrapper = shallow(<PrimeraApp saludo={saludo}/>);

        expect(wrapper).toMatchSnapshot();

    });

});