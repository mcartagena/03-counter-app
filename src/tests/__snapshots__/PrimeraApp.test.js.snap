// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp /> debe mostrar <PrimeraApp /> correctamente 1`] = `
<Fragment>
  <h1>
    Hola, soy Marcelo
  </h1>
  <p>
    soy un subtitulo
  </p>
</Fragment>
`;
